conference.js

```diff
@@ -1855,6 +1855,12 @@ export default {
             APP.UI.setAudioLevel(id, newLvl);
         });
 
+        room.on(JitsiConferenceEvents.TRACK_MUTE_CHANGED, (_, participantThatMutedUs) => {
+            if (participantThatMutedUs) {
+                APP.UI.notifyMuted(participantThatMutedUs);
+            }
+        });
```

Comment:
Could this be added somewhere else, maybe within the react files somewhere, so that more logic isn't added to conference.js? At the least I think we should avoid adding more to APP.UI.

Reply:
This was done while adding the mobile part.

Reply:
Not really, just the mute part, not the kick. I will update tomorrow and the kick part ...